Nodejs/Product/Npm/SPI/AbstractNpmLogSource.cs

```diff
@@ -1,10 +1,10 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
 
 namespace Microsoft.NodejsTools.Npm.SPI
 {
-    internal abstract class AbstractNpmLogSource : INpmLogSource
+    public abstract class AbstractNpmLogSource : INpmLogSource
```

Comment:
>public [](start = 4, length = 6)

Why did this change?

Reply:
NpmInstallCommand needs to be public in order to be accessed on the Nodejs assembly.

Reply:
The latest changes doesn't require this anymore. Rollback.